index.html

```diff
@@ -1814,30 +1814,29 @@ <h3>Status</h3>
               </li>
               <li>
 <code>type</code> <a>property</a>, which expresses the <a>credential</a> status
-type (also referred to as the <a>credential</a> status method).
+type.
               </li>
             </ul>
           </dd>
         </dl>
 
         <p>
-The precise contents of the <a>credential</a> status information is determined
+The precise content of the <a>credential</a> status information is determined
 by the specific <code>credentialStatus</code> <a>type</a> definition, and varies
 depending on factors such as whether it is simple to implement or if it is
-privacy-enhancing.
-It is expected that the value will provide enough information to determine the
-current status of the <a>credential</a> and that machine readable information
-needs to be retrievable from the URL. For example, the object could contain a
-link to an external document noting whether or not the <a>credential</a> is
-suspended or revoked.
+privacy-enhancing. It is expected that the value will provide enough information
+to determine the current status of the <a>credential</a> and that machine
+readable information will be retrievable from the URL. For example, the
+object could contain a link to an external document which notes whether or not the
+<a>credential</a> is suspended or revoked.
         </p>
 
         <pre class="example nohighlight"
           title="Usage of the status property">
 {
   "@context": [
     "https://www.w3.org/ns/credentials/v2",
-    "https://www.w3.org/ns/credentials/examples/v2"
+    "https://w3id.org/vc/status-list/2021/v1"
```

Comment:
Did you mean to remove the example context?  There's no v2 (yet), but v1 would still work here:
https://www.w3.org/2018/credentials/examples/v1
Without that context the input will expand the useful domain specific data `UniversityDegreeCredential`, `degree`, `BachelorDegree`, and `name` into the useless generic `https://www.w3.org/ns/credentials/issuer-dependent#` prefixed values.  I think the examples should better show how to create good data and avoid that vocab.